test/ng/compileSpec.js

```diff
@@ -7184,14 +7184,6 @@ describe('$compile', function() {
           "Please use the ng- versions (such as ng-click instead of onclick) instead.");
     }));
 
-    it('should pass through arbitrary values on onXYZ event attributes that contain a hyphen', inject(function($compile, $rootScope) {
```

Comment:
If I had to guess, that `on-` were valid attributes, although this test in itself is a bit weird since I don't think it should have had the opinionation of it testing interpolation.

The test conflicts with the new binding syntax though, since the expected behavior is to accept an expression as opposed to an interpolation.